lib/Transforms/Scalar/RewriteStatepointsForGC.cpp

```diff
@@ -69,7 +69,7 @@ namespace {
 struct RewriteStatepointsForGC : public FunctionPass {
   static char ID; // Pass identification, replacement for typeid
 
-  RewriteStatepointsForGC() : FunctionPass(ID) {
+  RewriteStatepointsForGC() : FunctionPass(ID), MustTrackBasePointers(true) {
```

Comment:
The createRewriteStatepointsForGCPass() initializes the MustTrackBasePointers field per client.
Was the RewriteStatepointsForGC constructor used outside of the createRewriteStatepointsForGCPass() somehow, such that initializing with default as true is necessary?